src/lib/View/SearchViewBuilder.php

```diff
@@ -82,6 +82,7 @@ public function buildView(array $parameters): SearchView
                 'results' => $this->pagerSearchContentToDataMapper->map($pagerfanta),
```

Comment:
Ensure that results are acquired before `getAggregations()` or `getSpellcheck()` call is made by PHP. I'd store this line in a variable before adding parameters.